hdbt.theme

```diff
@@ -454,6 +454,11 @@ function hdbt_preprocess_menu(&$variables) {
   foreach ($variables['items'] as &$item) {
     _hdbt_menu_item_apply_attributes($item, $apply_lang_attribute);
   }
+
+  // Menu, mobile menu, fallback menu and sidebar menu tree depth.
+  $menu_depth = 5;
```

Comment:
I'd explain this number a bit better here, a bit like it used to be before this PR removed the documentation
```
{# How deep do we allow the navigation menu to traverse? Counting from "instance", "main level", "level 3", "level 4", "level 5"... #}
```